src/commands/spawn.ts

```diff
@@ -21,6 +21,7 @@ export default class Spawn extends Command {
     grace: Flags.integer({ char: 'g', description: 'optional grace period (in seconds) to wait after last connection is closed before shutting down container' }),
     port: Flags.integer({ char: 'p', description: 'optional port for jamsocket to proxy requests to (default is 8080)' }),
     tag: Flags.string({ char: 't', description: 'optional tag for the service to spawn (default is latest)' }),
+    cluster: Flags.string({ char: 'c', description: 'optional cluster to spawn onto (default is jamsocket.run)'}),
```

Comment:
Perhaps we should add `hidden: true` to this flag to hide it from the docs / help command - at least until we have customers who have more than one cluster?

Also, I suppose creating a spawn token should accept the same `cluster` parameter?

Reply:
Agreed re. hiding it. Before adding it to the spawn token, I'd like to think through whether it should be a property of the token, the account, or the service.